### DISCUSSION TITLE
Reset clipboard

### DISCUSSION THREAD

**Discussion by jbarriojim** [mobile-dev-inc/Maestro#1604] [Q&A]
I'm using the command _copyTextFrom_ and after I'm using the _maestro.copiedText_ to access to the copied text. Now, if I copied other text with the same command the text is concatenated in a new line.

Then, my intention is try to empty the clipboard to copy the next text without the previous one. Some of my tries to empty it were:

`- evalScript: ${maestro.copiedText=""}` It seems that is not removing the previous value.

`- evalScript: ${maestro.copiedText=null}` It seems that is not removing the previous value.

`- clearState` It close the application

`- pasteText` I tried to use the paste action over no element, but the step is properly executed and the clipboard still with the previous value.

Do you have any idea about how to reset the clipboard of maestro.copiedText?

Thank you in advance.

**Comment by jbarriojim** [comment]
Ok, the problem was the next. Each time I console.log the variable, all the logs are printed:
![image](https://github.com/mobile-dev-inc/maestro/assets/24703132/91b46bda-3445-4236-acc5-0ec2a6b239ba)

But, the variable itself has the last value copied. It is not intuitive, but it is working 😄

**Comment by Fishbowler** [comment]
You're right, this logging is a bit misleading...

<img width="620" height="492" alt="image" src="https://github.com/user-attachments/assets/e23ac5a4-aec9-4df2-90fc-7e4907e826fb" />
